src/renderer/features/wallets/ForgetWallet/model/forget-wallet-model.ts

```diff
@@ -39,6 +90,34 @@ split({
   },
 });
 
+sample({
+  clock: [forgetWallet, wcDetailsModel.events.forgetButtonClicked],
+  source: {
+    wallets: walletModel.$wallets,
+    accounts: walletModel.$accounts,
+    proxies: proxyModel.$proxies,
+    walletsProxyGroups: proxyModel.$walletsProxyGroups,
+  },
+  fn: ({ wallets, accounts, proxies, walletsProxyGroups }, wallet) => ({
+    wallets,
+    accounts,
+    proxies,
+    walletsProxyGroups,
+    wallet,
+  }),
+  target: checkForProxiedWalletsFx,
+});
+
+sample({
+  source: checkForProxiedWalletsFx.doneData,
+  target: spread({
+    proxiesToDelete: proxyModel.events.proxiesRemoved,
+    proxiedWalletsToDelete: walletModel.events.walletsRemoved,
+    proxiedAccountsToDelete: balanceModel.events.accountsBalancesRemoved,
+    proxyGroupsToDelete: proxyModel.events.proxyGroupsRemoved,
```

Comment:
I think we should discuss when we should use `delete` and when `remove`

Reply:
Agree, thought about it a couple of days ago

Reply:
The discussion is only about adopting naming consistency across the code, do I understand it right?

Reply:
Right, it's about naming.
https://martinfowler.com/bliki/TwoHardThings.html